source-jdbc/src/main/java/org/apache/kylin/source/jdbc/JdbcHiveInputBase.java

```diff
@@ -222,11 +223,13 @@ protected AbstractExecutable createSqoopToFlatHiveStep(String jobWorkingDir, Str
 
             String splitTableAlias;
             String splitColumn;
+            String quoteFullNamedColumn; // `table.column`
             String splitDatabase;
             TblColRef splitColRef = determineSplitColumn();
             splitTableAlias = splitColRef.getTableAlias();
 
             splitColumn = getColumnIdentityQuoted(splitColRef, jdbcMetadataDialect, metaMap, true);
+            quoteFullNamedColumn = quoteIdentifier(partCol, jdbcMetadataDialect.getDialect()).toLowerCase();
```

Comment:
Use  `toLowerCase` is forbidden.
```
Forbidden method invocation: java.lang.String#toLowerCase() [Uses default locale]
[ERROR]   in org.apache.kylin.source.jdbc.JdbcHiveInputBase$JdbcBaseBatchCubingInputSide (JdbcHiveInputBase.java:232)
```
[ERROR] Scanned 34 (and 354 related) class file(s) for forbidden API invocations (in 0.10s), 1 error(s).

Reply:
fixed